CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Braintree iOS Drop-in SDK - Release Notes
 
+## unreleased
+
+* Remove deprecated methods and properties
+  * Remove BTDropInBaseViewController showLoadingScreen:animated: method
+  * Remove BTDropInRequest amount property
+  * Remove BTUIKAppearance darkTheme and lightTheme methods
```

Comment:
Can we add (NEXT_MAJOR_VERSION) next to each of these lines, so we don't accidentally release them in a minor version.